### ISSUE TITLE
Laravel Datatables Self Join Not Returning Joined Fields

### ISSUE THREAD

**Issue by dboeke** [yajra/laravel-datatables#577]
Trying to self-join the users table to retrieve the name of another user associated with the current user. but datatables seems to be ignoring the 2nd name column in the query.

in the controller
```php
$users = DB::table('users as u1')->join('users as u2', 'u2.coach_id', '=', 'u1.id')
            ->select([ 'u1.id', 'u1.name', 'u1.email', 'u1.coach_id', 'u2.name' ]);
return Datatables::of($users)->make(true);
```

javascript
```js
columns: [
            { data: 'id', name: 'u1.id' },
            { data: 'name', name: 'u1.name' },
            { data: 'email', name: 'u1.email' },
            { data: 'name', name: 'u2.name' },
        ]
```

I have looked at the returning json and the field does not show up under the "data" collection, but when i look at the query that was run, the query is correct and returns both name fields....

### System details

- Ubuntu 14.04.3 
- PHP Version 7.0.1
- Laravel Version 5.1
- Laravel-Datatables Version 6.11

**Comment by dboeke** [comment]
Solved this on my own.  Fix below:

in the controller
```php
$users = DB::table('users as u1')->join('users as u2', 'u1.coach_id', '=', 'u2.id')
            ->select([ 'u1.id', 'u1.name', 'u1.email', 'u1.coach_id', 'u2.name as coach' ]);
return Datatables::of($users)->make(true);
```

javascript
```js
columns: [
            { data: 'id', name: 'u1.id' },
            { data: 'name', name: 'u1.name' },
            { data: 'email', name: 'u1.email' },
            { data: 'coach', name: 'u2.name' },
        ]
```

**Comment by yajra** [comment]
Fixed via #2051.
